### PR TITLE
Fix ctrl+q parity remove old not working code.

### PR DESCRIPTION
**Pull request by Nirad**
fix for issues #55 
Now ctrl+q work like Original Client clean up old code

### CODE REVIEW

File: src/TextEngine/GameConsole.cpp [hunk 1]
```diff
@@ -367,14 +367,12 @@ void CGameConsole::DrawW(
 void CGameConsole::SaveConsoleMessage()
 {
     DEBUG_TRACE_FUNCTION;
-    if (m_ConsoleStack[m_ConsoleSelectedIndex % MAX_CONSOLE_STACK_SIZE] != Text)
+
+    m_ConsoleStack[m_ConsoleStackCount % MAX_CONSOLE_STACK_SIZE] = Text;
```
Comment (fungos) [inline src/TextEngine/GameConsole.cpp]: Is this right? We will be able to save repeated message? I'll need to check on the original client too.

Comment (Nirad) [reply src/TextEngine/GameConsole.cpp]: yes all message in original client are save even repeated one ....

Comment (Nirad) [reply src/TextEngine/GameConsole.cpp]: but if you want we can keep these as a feature of CrossUO ...

Comment (fungos) [reply src/TextEngine/GameConsole.cpp]: I don't think it is worth as a feature anyway, lets be more parity than custom.

**Pull request merged**